lib/Backend/Sym.cpp

```diff
@@ -1090,9 +1091,14 @@ Sym::Dump(IRDumpFlags flags, const ValueType valueType)
                             uint index = stackSym->GetByteCodeRegSlot() - stackSym->GetByteCodeFunc()->GetJITFunctionBody()->GetConstCount();
                             Js::PropertyId propertyId = functionBody->GetPropertyIdOnRegSlotsContainer()->propertyIdsForRegSlots[index];
                             Output::Print(_u("(%s)"), stackSym->GetByteCodeFunc()->GetInProcThreadContext()->GetPropertyRecord(propertyId)->GetBuffer());
+                            printed = true;
                         }
                     }
                 }
+                if (!printed)
+                {
+                    Output::Print(_u("{%d}"), stackSym->GetByteCodeRegSlot());
```

Comment:
Good point; I've removed the added debug code for the time being. It might be useful to add something like this in the future though (under an additional flag, perhaps), in order to aid in diagnosis of bytecodereg lifetime issues.